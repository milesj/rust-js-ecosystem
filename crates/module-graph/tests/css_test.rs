mod utils;

use starbase_sandbox::{assert_snapshot, create_sandbox};
use utils::generate_graph_for_file;

mod css {
    use super::*;

    #[test]
    fn normal() {
        let sandbox = create_sandbox("css");

        assert_snapshot!(generate_graph_for_file(sandbox.path(), "styles.css"));
    }

    #[test]
    fn modules() {
        let sandbox = create_sandbox("css");

        assert_snapshot!(generate_graph_for_file(sandbox.path(), "styles.module.css"));
    }
}
