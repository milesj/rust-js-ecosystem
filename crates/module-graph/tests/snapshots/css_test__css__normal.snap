---
source: crates/module-graph/tests/css_test.rs
expression: "generate_graph_for_file(sandbox.path(), \"styles.css\")"
---
{
    1: Module {
        exports: [],
        fragment: None,
        id: 1,
        imports: [],
        package_name: None,
        path: "/root/styles.css",
        query: None,
        source: CssModule {
            exports: {},
            source: "@import \"./other.css\";\n\n.container {\n    display: flex;\n}\n\n.button {\n    background-color: black;\n}\n\n.button:hover {\n    background-color: white;\n}\n",
        },
    },
}
